Solutions/Australian Cyber Security Centre/SolutionMetadata.json

```diff
@@ -0,0 +1,15 @@
+{
+	"publisherId": "azuresentinel",
+	"offerId": "azure-sentinel-solution-australiancybersecurity",
+	"firstPublishDate": "2022-11-23",
+	"providers": ["Australian Cyber Security Centre"],
+	"categories": {
+		"domains" : ["Security – Threat Intelligence"]
```

Comment:
Please check if we need to add "Security - Automation (SOAR)" as well, thanks.

Reply:
Prateek only shared ["Security - Threat Intelligence"] domain category. Do you want me to check with Prateek once.
Thanks